services/madoc-ts/src/frontend/themes/helpers/themeVariable.ts

```diff
@@ -18,32 +18,3 @@ export function themeVariable<
 
   return theme(name, newDef);
 }
-
-export function themeVariables<
-  Key extends keyof ThemeDefinitions,
-  Keys extends keyof ThemeDefinitions[Key],
-  Definition extends ThemeDefinitions[Key],
-  Definitions extends Omit<
-    {
-      [Def in Keys]?: ThemeDefinitionMap<Key, Def>;
-    },
-    '__VARIATIONS__'
-  >
->(
-  name: Key,
-  definition: Definitions
-): Omit<
-  {
-    [Def in keyof Definitions]: theme.ThemeSet;
-  },
-  '__VARIATIONS__'
-> {
-  const map: any = {};
-  const keys = Object.keys(definition);
-
-  for (const key of keys) {
-    map[key] = themeVariable(name, key as any, (definition as any)[key]);
-  }
-
-  return map;
-}
```

Comment:
This is a short hand that's not been used yet. Currently you can create a theme variables for stylesheets.

Theme variables lets you define a whole bunch of variables at once:
```js
const test = themeVariables('accent', {
  primary: {
    default: 'white'
  },
  secondary: {
    default: 'gray'
  }
});

// test.primary
// test.secondary
```

Which is the same as the current helper, but with less code:
```js
const primary = themeVariable('accent', 'primary', {
  default: 'white',
});
const secondary = themeVariable('accent', 'secondary', {
  default: 'gray',
});
```